client/src/components/Feed/MessagesModal/MessageModal.js

```diff
@@ -68,7 +68,7 @@ const MessageModal = ({
       showModal();
     }
     sessionStorage.removeItem("msgModal");
-  }, [])
+  }, [authorId, isAuthenticated]);
```

Comment:
This should not be added here 😅
I assume it was added by the linter.
Please clear the dependency array and add the eslint ignore rule.